Cannot find module 'semver'

Hello, 
after upgrading to 0.23 i ran into that error on a lambda.

```Error
{
   "errorType":"Runtime.ImportModuleError",
   "errorMessage":"Error: Cannot find module 'semver'\nRequire stack:\n- /var/task/node_modules/nexus-plugin-prisma/dist/index.js\n- /var/task/src/server.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
   "stack":[
      "Runtime.ImportModuleError: Error: Cannot find module 'semver'",
      "Require stack:",
      "- /var/task/node_modules/nexus-plugin-prisma/dist/index.js",
      "- /var/task/src/server.js",
      "- /var/runtime/UserFunction.js",
      "- /var/runtime/index.js",
      "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
      "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
      "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
      "    at Module._compile (internal/modules/cjs/loader.js:1137:30)",
      "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)",
      "    at Module.load (internal/modules/cjs/loader.js:985:32)",
      "    at Function.Module._load (internal/modules/cjs/loader.js:878:14)",
      "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)",
      "    at internal/main/run_main_module.js:17:47"
   ]
}
```

Maybe related to https://github.com/graphql-nexus/nexus-plugin-prisma/commit/5d8cc650c3d353305cba4a28b376ba71796cb9e3 ?

Versions:

- nexus-plugin-prisma: 0.23.0
- @prisma/client: 2.10.0
- @nexus/schema: 0.17.0

--- a/package.json
+++ b/package.json
@@ -41,7 +41,8 @@
     "fs-jetpack": "^4.0.0",
     "lodash": "^4.17.20",
     "outdent": "^0.7.1",
-    "pluralize": "^8.0.0"
+    "pluralize": "^8.0.0",
+    "semver": "^7.3.2"
   },
   "devDependencies": {
     "@nexus/schema": "0.15.0",
@@ -71,7 +72,6 @@
     "node-fetch": "2.6.1",
     "prettier": "2.0.5",
     "rimraf": "3.0.2",
-    "semver": "7.3.2",
     "strip-ansi": "6.0.0",
     "ts-jest": "26.4.1",
     "ts-morph": "8.1.2",

--- a/src/index.ts
+++ b/src/index.ts
@@ -45,7 +45,7 @@ function ensurePeerDepRangeSatisfied(depName: string) {
       return
     }
 
-    if (semver.satisfies(supportedRange, installedVersion)) {
+    if (semver.satisfies(installedVersion, supportedRange)) {
       return
     }
 

--- a/src/index.ts
+++ b/src/index.ts
@@ -61,8 +61,11 @@ ensureDepIsInstalled('@nexus/schema')
 ensureDepIsInstalled('graphql')
 ensureDepIsInstalled('@prisma/client')
 
-ensurePeerDepRangeSatisfied('@nexus/schema')
-ensurePeerDepRangeSatisfied('graphql')
+// TODO: Bring back peer dep range check for @nexus/schema and graphql once we have proper ranges
+// TODO: They're currently way too conservative
+
+//ensurePeerDepRangeSatisfied('@nexus/schema')
+//ensurePeerDepRangeSatisfied('graphql')
 ensurePeerDepRangeSatisfied('@prisma/client')
 
 export * from './plugin'

